templates/4a_compensate_image_data.ipynb

```diff
@@ -5,9 +5,26 @@
    "id": "ca8b60ef",
```

Comment:
<p>great, looks like all the testing presets are up at the top! you can just put all five in one cell here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>